scenarioo-client/app/build/scenarios-overview/scenarios-overview.component.ts

```diff
@@ -98,6 +99,8 @@ export class ScenariosOverviewComponent {
             .subscribe(((labelConfigurations: LabelConfigurationMap) => {
                 this.labelConfigurations = labelConfigurations;
             }));
+
+        this.isPanelCollapsed = this.localStorageService.getBoolean('scenarioo-metadataVisible-useCaseView', false);
```

Comment:
I would like to see this code inside the detailsarea component. And not have it on each page that uses the component.

Reply:
you can put it on your list to resolve later